slick/src/main/scala/slick/jdbc/JdbcActionComponent.scala

```diff
@@ -208,17 +203,27 @@ trait JdbcActionComponent extends SqlActionComponent { self: JdbcProfile =>
         case ParameterSwitch(cases, default) =>
           findSql(cases.find { case (f, n) => f(param) }.map(_._2).getOrElse(default))
       }
-      (tree match {
+      val (comment, tree2) = tree match {
+        case Comment(comment, child) =>
+          if(comment.isEmpty) {
+            ("", child)
+          } else {
+            (s"/* $comment */\n", child)
```

Comment:
This substitution is vulnerable to SQL injection. For example, if the value of `comment` is `"*/ DELETE FROM projects; /*"` then our `projects` table gets wiped when the query is run. Maybe this isn't a huge concern for us, since our comments will be automatically generated and don't come from input data. But if this is going to be a general-purpose feature then we should be escaping the comments or making them harmless.

Reply:
Yes. I agree. I can add this in the details of the ticket which includes creating a PR to the upstream slick repo